ml-agents-envs/mlagents_envs/side_channel/stats_side_channel.py

```diff
@@ -0,0 +1,48 @@
+from mlagents_envs.side_channel import SideChannel, IncomingMessage
+import uuid
+from typing import Dict, Tuple
+from enum import Enum
+
+
+# Determines the behavior of how multiple stats within the same summary period are combined.
+class StatsAggregationMethod(Enum):
```

Comment:
Is there a better place for this? It needs to live in ml-agents-envs (unless we move the SideChannel?) but it feels strange importing `mlagents_envs.side_channel.stats_side_channel` in a lot of places just to get the enum. @ervteng thoughts?

Reply:
Hmm, this actually seems the least obtrusive since the StatsReporter lives in `mlagents`. The other option would be in the `stats.py`, but then we'd have to move that to `mlagents_envs`.